woocommerce/woocommerce-ios
# Fix Gravatar image to be circular

Closes: https://github.com/woocommerce/woocommerce-ios/issues/3366.
Depends on https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/638.

## Description

This PR updates WordPressAuthenticator lib to version with a fix applied in `xib` file.
For details check https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/627.

There are additional changes introduced in 1.42.0 -> 1.42.3:
- https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/629
  - No more warning from 1PasswordExtension!
- https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/630
- https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/632

## Testing

1. Logout from the app.
2. Tap "Continue with WP.com".
3. Enter your email and tap "Continue".
4. Confirm that avatar image is circular.

## Screenshots

before | after
--|--
![Simulator Screen Shot](https://user-images.githubusercontent.com/3132438/145080454-bc9fc2e8-8fea-4331-86ef-fbf196521836.png)|![Simulator Screen Shot](https://user-images.githubusercontent.com/3132438/145079989-5538d780-2417-4619-974b-66f2d3b260db.png)

---
